library/std/src/os/freebsd/fs.rs

```diff
@@ -137,6 +144,10 @@ impl MetadataExt for Metadata {
         self.as_inner().as_inner().st_flags as u32
     }
     fn st_lspare(&self) -> u32 {
+        #[cfg(freebsd12)]
+        panic!("st_lspare not supported with FreeBSD 12 ABI");
+        #[cfg(not(freebsd12))]
+        #[cfg(not(freebsd12))]
         self.as_inner().as_inner().st_lspare as u32
     }
```

Comment:
Unfortunately, in this case the rustfmt fix suggestion is bogus. Following should work:

```suggestion
    #[cfg(freebsd12)]
    fn st_lspare(&self) -> u32 {
        panic!("st_lspare not supported with FreeBSD 12 ABI");
    }
    #[cfg(not(freebsd12))]
    fn st_lspare(&self) -> u32 {
        self.as_inner().as_inner().st_lspare as u32
    }
```